### PR TITLE
Follower/Followed Search and paginate

### PR DESCRIPTION
**Pull request by nil1511**

### CODE REVIEW

File: src/user/UserList.js [hunk 1]
```diff
@@ -1,18 +1,50 @@
 import React from 'react';
 import { Link } from 'react-router';
+import ReduxInfiniteScroll from 'redux-infinite-scroll';
+import _ from 'lodash';
```
Comment (adcpm) [inline src/user/UserList.js]: Avoid using root better to use `import { isNumber, filter, take } from 'lodash';`

Comment (nil1511) [inline src/user/UserList.js]: Udpated in d787625

File: src/user/UserList.js [hunk 2]
```diff
@@ -1,18 +1,50 @@
...
-      <div className="UserList pvm">
-        <Avatar md username={this.props.username} className="mrm" />
-        <Link to={`/@${this.props.username}`}>@{this.props.username}</Link>
-      </div>
-    );
+    const filterText = this.state.filterText;
+    const defaultPageItems = 10;
+    const noOfItemsToShow = defaultPageItems * this.state.page;
+    const users = this.state.filterText ?
+      filter(this.props.users, user => user.indexOf(filterText) >= 0) : this.props.users;
```
Comment (p0o) [inline src/user/UserList.js]: why not using the Array.filter() in-built function of javascript?

File: src/user/UserList.js [hunk 3]
```diff
@@ -1,18 +1,50 @@
...
+
+    return (<div>
```
Comment (p0o) [inline src/user/UserList.js]: It's nicer to start jsx the next line after paranthesis:

```
return (
  <div>
    ..
  </div>
);

File: src/user/UserList.js (hunk 2 above)
Comment (nil1511) [inline src/user/UserList.js]: Array.filter might not be present in some older browsers.

**Pull request merged**